src/webots/gui/WbUpdatedDialog.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 1996-2021 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "WbUpdatedDialog.hpp"
+
+WbUpdatedDialog::WbUpdatedDialog(QWidget *parent) : QDialog(parent) {
+  setWindowTitle(tr("Welcome to Webots 2022a"));
+
+  QPixmap webotsLogo("images:webots.png");
+  QLabel *mImage = new QLabel(this);
+  mImage->setGeometry(QRect(16, 15, 50, 50));
+  mImage->setPixmap(webotsLogo);
+  mImage->setScaledContents(true);
+
+  mLabelThanks = new QLabel(this);
+  mLabelThanks->setGeometry(QRect(70, 30, 330, 31));
+  mLabelThanks->setText(tr("<b>Thank you for updating to the last version of Webots.</b>"));
+  mLabelThanks->setAlignment(Qt::AlignLeading | Qt::AlignLeft | Qt::AlignTop);
+  mLabelThanks->setWordWrap(true);
+
+  mGroupBox = new QGroupBox(this);
+  mGroupBox->setGeometry(QRect(35, 65, 365, 230));
+  mGroupBox->setTitle(tr("Important Note for previous users"));
+  mGroupBox->setStyleSheet("QGroupBox {border: 1px solid gray;border-radius: 9px;margin-top: 0.5em; } "
+                           "QGroupBox::title {subcontrol-origin:  margin; subcontrol-position: top center; }");
+
+  mLabelNote = new QLabel(mGroupBox);
+  mLabelNote->setGeometry(QRect(15, 20, 345, 200));
+  mLabelNote->setText(
+    tr("<p style=\"line-height:1.2\"> <br />"
+       "Webots axis systems and objects orientation conventions respectively <b>changed to ENU and FLU</b>. All the PROTOs and "
+       "worlds included in the Webots package are now in ENU and FLU. <br /> <br /> <b>If you are using your own world or "
+       "PROTO</b>, "
+       "Webots will "
+       "rotate all the elements of your world when you open it in this new version. However, you may have to rotate some "
```

Comment:
```suggestion
       "All Webots worlds are now in the ENU axis system and all Webots devices, geometries, and PROTOs are in the FLU axis system. If your world or PROTO includes affected devices, geometries, or PROTOs then Webots will try to preserve the old behavior by rotating them. However, you may have to rotate some "
```
Feel free to improve it, but this is what I was trying to change:
- The first two sentences can be simplified into a single one.
- Webots will not rotate all elements, but only nodes affected by the change.
- Not only PROTOs are FLU, but also devices and geometries.
- We have to explain to the users why Webots rotates the nodes. From the text, I got the impression that Webots rotates the nodes to get ENU/FLU axis orientation.